client/Makefile

```diff
@@ -1,5 +1,7 @@
 include ../includes.mk
 
+.PHONY: build
```

Comment:
let's remove this. Just remove the `client/build` folder locally and everything should work fine again. :)